src/Commands/DocumentFacades.php

```diff
@@ -30,51 +33,52 @@ class DocumentFacades extends Command
      */
     public function handle(): int
     {
-        $facades = $this->facades();
+        $this->info('Generating document annotations...');
 
-        if ($only = $this->option('only')) {
-            $facades = $facades->filter(
-                fn (string $classname) => in_array($classname, $only)
-            );
-        } elseif ($except = $this->option('except')) {
-            $facades = $facades->filter(
-                fn (string $classname) => ! in_array($classname, $except)
-            );
-        }
+        $result = Process::run(sprintf(
+            'php -f vendor/bin/facade.php -- %s',
+            $this->facades()
+                ->map(fn (string $class) => str_replace('\\', '\\\\', $class))
+                ->join(' ')
+        ));
+
+        if ($result->failed()) {
+            $this->error($result->output());
 
-        $result = Process::run(sprintf('php -f vendor/bin/facade.php -- %s', $facades->map(
-            fn (string $classname) => str_replace('\\', '\\\\', $classname)
-        )->join(' ')));
+            return self::FAILURE;
+        }
 
-        $result->successful()
-            ? $this->info($result->output())
-            : $this->error($result->output());
+        $this->info($result->output());
 
-        return $result->successful()
-            ? static::SUCCESS
-            : static::FAILURE;
+        return self::SUCCESS;
     }
 
     /**
-     * Get the application's facades.
+     * Get a file from base path.
      */
     protected function facades(): Collection
     {
-        return collect(
-            Discover::in(...$this->paths())
-                ->classes()
-                ->extending(Facade::class)
-                ->get()
-        );
+        $except = $this->option('except');
+        $only = $this->option('only');
+
+        return collect(Finder::create()->files()->depth(0)->in($this->argument('path'))->name('*.php'))
```

Comment:
This needs to support searching multiple paths, like was previously supported with Spatie Structure Discovery 👍